backend/poetry.lock

```diff
@@ -159,76 +151,73 @@ files = [
 
 [[package]]
 name = "pyyaml"
-version = "6.0"
+version = "6.0.1"
```

Comment:
pyyam upgraded here :point_up:

Reply:
YAAAASSSS!!!

Reply:
oh... no...
https://github.com/hackforla/CivicTechJobs/actions/runs/5907278236/job/16089298214#step:5:227
pyyaml is still not updating...